src/test/java/com/exasol/drivers/JdbcDriverTest.java

```diff
@@ -87,14 +88,27 @@ void testToString() {
 
     @Test
     void testToStringWithSourceFile() {
-        assertThat(JdbcDriver //
-                .builder("the_name") //
-                .prefix("jdbc:the_prefix") //
-                .sourceFile(Path.of("/the/path")) //
-                .mainClass("com.example.Driver") //
-                .build() //
-                .toString(), //
-                equalTo("JDBC driver \"the_name\" (com.example.Driver), source: \"/the/path\""));
+        final String osName = System.getProperty("os.name");
+        final String windowsStr = "Windows";
+        if (StringUtils.containsIgnoreCase(osName, windowsStr)) {
+            assertThat(JdbcDriver //
+                    .builder("the_name") //
+                    .prefix("jdbc:the_prefix") //
+                    .sourceFile(Path.of("/the/path")) //
+                    .mainClass("com.example.Driver") //
+                    .build() //
+                    .toString(), //
+                    equalTo("JDBC driver \"the_name\" (com.example.Driver), source: \"\\the\\path\""));
+        } else {
+            assertThat(JdbcDriver //
+                    .builder("the_name") //
+                    .prefix("jdbc:the_prefix") //
+                    .sourceFile(Path.of("/the/path")) //
+                    .mainClass("com.example.Driver") //
+                    .build() //
+                    .toString(), //
+                    equalTo("JDBC driver \"the_name\" (com.example.Driver), source: \"/the/path\""));
+        }
```

Comment:
I tink there's a simpler solution:

```suggestion
        assertThat(JdbcDriver //
                .builder("the_name") //
                .prefix("jdbc:the_prefix") //
                .sourceFile(Path.of("/the/path")) //
                .mainClass("com.example.Driver") //
                .build() //
                .toString(), //
                startsWith("JDBC driver \"the_name\" (com.example.Driver), source:"));
```